addons/sales_team/models/crm_team.py

```diff
@@ -170,12 +170,12 @@ def _compute_member_warning(self):
                 ('user_id', 'in', team.member_ids.ids)
             ])
             if other_memberships:
-                member_warning = _("Adding %(user_names)s in this team will remove them from %(team_names)s.",
+                member_warning = _("%(user_names)s already in other teams (%(team_names)s).",
                                    user_names=", ".join(other_memberships.mapped('user_id.name')),
                                    team_names=", ".join(other_memberships.mapped('crm_team_id.name'))
                                   )
             if member_warning:
-                team.member_warning = member_warning + " " + _("Working in multiple teams? Activate the option under Configuration>Settings.")
+                team.member_warning = member_warning + " " + _("Working in multiple teams?")
```

Comment:
Well then I would remove the "Working in multiple teams" ? as it make the result strange ... something is missing :) . With the button it makes sense, but the warning by itself it a bit weird that way.

Warning is simpler if we remove the last part, then either we add it somehow in form view, either just "Activate option" should be sufficient. Could you check ?